pom.xml

```diff
@@ -144,6 +144,17 @@
     </pluginManagement>
   </build>
 
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>pl.ds.websight</groupId>
```

Comment:
as I understand this is needed only to make the mvnw versions:update-properties command used in pipieline to work correctly, eight? if yes, then please add comment with information